### ISSUE TITLE
build: Verify exports in package.json

### ISSUE THREAD

**Issue by gpbl** [gpbl/react-day-picker#2533]
We are receiving some reports (e.g. https://github.com/gpbl/react-day-picker/discussions/2521), about exports not being correctly interpreted by some bundler:

https://arethetypeswrong.github.io/?p=react-day-picker%409.1.3

### Problems

- [ ] [Resolution failed](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/NoResolution.md)
Import failed to resolve to type declarations or JavaScript files.
- [ ] [ESM (dynamic import only)](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/CJSResolvesToESM.md)
A require call resolved to an ESM JavaScript file, which is an error in Node and some bundlers. CommonJS consumers will need to use a dynamic import.
- [ ] [Used fallback condition](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FallbackCondition.md)
Import resolved to types through a conditional package.json export, but only after failing to resolve through an earlier condition. This behavior is a [TypeScript bug](https://github.com/microsoft/TypeScript/issues/50762). It may misrepresent the runtime behavior of this import and should not be relied upon.
- [ ] [Internal resolution error](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/InternalResolutionError.md)
Import found in a type declaration file failed to resolve. Either this indicates that runtime resolution errors will occur, or (more likely) the types misrepresent the contents of the JavaScript files.
Expand the raw details below to see the imports that failed to resolve.

**Comment by Sandip-Z** [comment]
I am still having this issue after updating to v9.1.4

**Comment by JamersonAguiarOSF** [comment]
Still having the issue also

**Comment by gpbl** [comment]
Hi @Sandip-Z @JamersonAguiarOSF

Please help by reproducing your problem on [CodeSandbox](https://codesandbox.io) or [StackBlitz](https://stackblitz.com) and then create a new issue.

Thanks!

**Comment by khinshankhan** [comment]
This may be irrelevant but I found that my jest tests couldn't import the stylesheets. Using `identity-obj-proxy` as guided in [this stackoverflow answer](https://stackoverflow.com/a/50572999) worked.

Just commenting in case someone else ends up here while searching the exact error like I did 😅

**Comment by JamersonAguiarOSF** [comment]
> This may be irrelevant but I found that my jest tests couldn't import the stylesheets. Using `identity-obj-proxy` as guided in [this stackoverflow answer](https://stackoverflow.com/a/50572999) worked.
> 
> Just commenting in case someone else ends up here while searching the exact error like I did 😅

I'll give it a try, thank you
